src/Tizen.NUI.Components/Controls/Loading.cs

```diff
@@ -36,14 +36,30 @@ public class Loading : Control
             var instance = (Loading)bindable;
             if (newValue != null)
             {
-                instance.loadingStyle.Images = (string[])newValue;
+                instance.loadingStyle.ImageList = new List<string>((string[])newValue);
                 instance.imageVisual.URLS = new List<string>((string[])newValue);
             }
         },
         defaultValueCreator: (bindable) =>
         {
             var instance = (Loading)bindable;
-            return instance.loadingStyle.Images;
+            return instance.loadingStyle.ImageList.ToArray();
+        });
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
```

Comment:
We may not open bindable properties. Please remove wrong comment.

Reply:
done